main.js

```diff
@@ -300,14 +336,40 @@ var $checkout = document.querySelector('[data-view="checkout"]')
 renderAppState(app)
 
 $catalog.addEventListener('click', function (event) {
+  var $target = event.target
   var $selectedCard = event.target.closest('[itemID]')
   if ($selectedCard !== null) {
     var idNum = $selectedCard.getAttribute('itemID')
     idNum = parseInt(idNum, 10)
     app.view = 'details'
     app.details.item = findItem(app.catalog.items, idNum)
+    renderAppState(app)
+  }
+  if ($target.getAttribute('id') === 'dropdown-button') {
+    var $dropItems = $target.nextSibling.querySelectorAll('a')
+    $dropItems.forEach(function (element) {
+      element.classList.toggle('d-none')
+      element.classList.toggle('d-block')
+    })
+  }
+  if ($target.getAttribute('id') === 'high') {
```

Comment:
It would be more usable if you could explicitly choose _not_ to sort from the dropdown.